packages/teams-js/src/private/videoEx.ts

```diff
@@ -117,38 +173,63 @@ export namespace videoEx {
    * Register to process video frames
    * @beta
    *
-   * @param frameCallback - The callback to invoke when registerForVideoFrame has completed
-   * @param config - VideoFrameConfig to customize generated video frame parameters
+   * @param parameters - Callbacks and configuration to process the video frames. A host may support either {@link VideoFrameHandler} or {@link VideoBufferHandler}, but not both.
+   * To ensure the video effect works on all supported hosts, the video app must provide both {@link VideoFrameHandler} and {@link VideoBufferHandler}.
+   * The host will choose the appropriate callback based on the host's capability.
    *
    * @internal
    * Limited to Microsoft-internal use
    */
   export function registerForVideoFrame(parameters: RegisterForVideoFrameParameters): void {
-    ensureInitialized(runtime, FrameContexts.sidePanel);
-    if (!isSupported()) {
```

Comment:
Why are we removing the isSupported check?

Reply:
I will be more explicit: you can't remove the isSupported check.

Reply:
`isSupport` check is not actually removed, the if-else branches below this line already covered all the supported/unsupported cases. 
I'll add it back if you prefer explicit checks.